Bearded.Utilities.Tests/Algorithms/BinPackingTests.cs

```diff
@@ -18,7 +18,8 @@ public void TestPackedRectangles_IncludesAll()
 
             var result = BinPacking.Pack(input);
 
-            Assert.Equal(100, result.Rectangles.Count);
+            Assert.NotNull(result);
+            Assert.Equal(100, result!.Rectangles.Count);
```

Comment:
Optional:

Have you considered using FluentAssertions if you are rewriting these anyway?

Reply:
Happy to do so if you like, I wanted to keep the changes to a minimum.

Reply:
We can follow up separately.